libraries/bootstrap.php

```diff
@@ -41,6 +41,12 @@ class_exists('\\Joomla\\CMS\\Autoload\\ClassLoader');
 // Decorate Composer autoloader
 spl_autoload_register([new \Joomla\CMS\Autoload\ClassLoader($loader), 'loadClass'], true, true);
 
```

Comment:
```suggestion

// Populate $_ENV if it's disabled by variables_order in php.ini
if (empty($_ENV)) {
    // getenv() is not thread-safe and it can cause segmentaion fault, so we should try $_SERVER first
    $_ENV = !empty($_SERVER) ? array_filter($_SERVER, fn ($key) => str_starts_with($key, 'JOOMLA_'), ARRAY_FILTER_USE_KEY) : getenv();
}

```
I've just checked [Homebrew's php](https://github.com/Homebrew/homebrew-core/blob/826c3fd55f451bfea3efe58d178bf60b1e690774/Formula/p/php.rb#L224) installation. The `$_ENV` is disabled by default. So, in Mac OS the `$_ENV` won't work without tweaking php.ini. I suggest to populate `$_ENV` manually.

Reply:
I will look in to it, thanks